README.md

```diff
@@ -1,7 +1,108 @@
-# MIGRATOR.py
+# Package for applying migrations to the postgresql database
+
+[![Lint](https://github.com/Drozd0f/migrator/actions/workflows/linter.yml/badge.svg)](https://github.com/Drozd0f/migrator/actions/workflows/linter.yml)
 
 Not for production purposes.
 
 Tool to migrate database from files.
 
-[![Lint](https://github.com/qerdcv/migrator/actions/workflows/linter.yml/badge.svg)](https://github.com/qerdcv/migrator/actions/workflows/linter.yml)
+## **Install**
+
+```shell
+$ pip install pgrate
+```
+
+## **Usage**
+
+```shell
+pgrate -p directorie/to/migrations -d postgres://username:password@localhost:5432/database
+```
+
+**OR**
+
+```shell
+pgrate -p directorie/to/migrations -d postgresql://username:password@localhost:5432/database
+```
+
+> **NOTE**
+> Migration name must start with a number + "_" and end with *.up.sql example (001_migration_name.up.slq)
```

Comment:
Migration name must start with a number + "_", where number is migration version*